### DISCUSSION TITLE
HTTP2 for custom domains

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#145] [Anything Else]
"I noticed custom domains /js/index.js is served using http/1.1. Is it feasible for this to be served using HTTP2 instead?"

([imported from the old roadmap](https://plausible.nolt.io/104))

**Comment by rafalkukla** [comment]
Hiya, noticed this today as well, on self hosted - any update on serving /js/plausible.outbound-links.js over HTTP2 instead?

**Comment by metmarkosaric** [comment]
hi @rafalkukla! we no longer recommend custom domains as we now recommend a proxy so i will close this: https://plausible.io/docs/proxy/introduction
